analiza-rozwiazan-biblioteki-komponentow-GUI.rst

```diff
@@ -0,0 +1,330 @@
+   typu odbiorców)
+
+.. figure:: images/standardy-projektowe-1.png
+
+.. figure:: images/standardy-projektowe-2.png
+
+.. figure:: images/standardy-projektowe-3.png
+
+Inne inicjatywy rządowe w zakresie transformacji cyfrowej:
+
+-  Norwegia – https://www.norge.no/
+-  Nowa Zelandia – https://www.govt.nz/
+-  Hong Kong – http://www.gov.hk/en/residents/
+-  Singapur – https://www.ecitizen.gov.sg/Pages/default.aspx
+-  Estonia – https://e-estonia.com/
+
+Efekt analizy - przekrojowe Standardy Projektowania
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Głównym rezultatem analizy jest wniosek, że biblioteka komponentów GUI
+to tylko czubek góry lodowej. Dobrze zaprojektowany system wymaga
+instrukcji obejmującej większy zakres informacji niż tylko listę
+dostępnych klocków. Kompletny zbiór reguł ich używania powinien
+obejmować następujące obszary:
+
+Wytyczne dla identyfikacji wizualnej:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Część opisująca style dla elementów graficznych GUI
+
+-  Paleta kolorów
+-  Posługiwanie się znakiem / znakami
+-  Typografia
+-  Antywzorce
+-  Przykładowe wdrożenia
+-  Ikony
+
+Zasady projektowania UI dla interfejsów responsywnych:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Część opisująca filozofię projektowania na podstawie głównych zagadnień
+
+-  Biblioteka komponentów
+-  Layouty
+-  Grid
+-  Nawigacja
+-  Motion
+-  Mikrointerakcje
+-  Dostępność
+-  Walidacja
+-  Pomoc i dodatkowa wiedza
+-  Notyfikowanie
+-  Przykładowe wdrożenia
+
+Komunikacja z użytkownikami:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Część opisująca sposoby, styl i ton komunikacji pomiędzy systemem a
+użytkownikami
+
+-  Jak tworzyć komunikaty do użytkowników
+-  Jakiego języka używać, a jakiego nie używać.
+-  Jakie formy komunikacji można stosować (mailing, dokumenty …)
+
+Zasady kodowania:
+^^^^^^^^^^^^^^^^^
+
+Część opisująca stosowanie nakładek i reguły tworzenia nowych
+
+-  Dobre praktyki pisania kodu
+-  Dobre praktyki organizowania projektu od strony DEV
+-  Zasady publikowania na Github
+-  Informacje o licencji
+
+Korzyści i ryzyka wynikające z wdrożenia przekrojowych Standardów Projektowych
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Korzyści:
+^^^^^^^^^
+
+-  Dysponując powyższym zbiorem wykonawca będzie miał kompletny zbiór
+   instrukcji odpowiadający na większość pytań dotyczących
+   charakterystyki realizowanej aplikacji.
+-  Wdrożenie może być jeszcze tańsze, gdyż część pracy po stronie
+   wykonawcy będzie zawarta w opisie standardów.
+-  Aplikacje w marketplace będą miały spójny charakter, wygląd i
+   nawigację, co skróci czas uczenia się nowych aplikacji.
+-  Łączenie aplikacji między sobą będzie prostsze i bardziej
+   przewidywalne.
+-  Standardy mogą zostać powielone na systemy frontowe, czyli wszystkie
+   te, które będą dostępne dla obywateli (na wzór gov.uk i innych), co
+   stanowiłoby duży krok w kierunku przemyślanej transformacji cyfrowej.
```

Comment:
Czy nie jest to miejsce na odesłanie do unifikacji stron ministerstw, a także docelowo szerszej unifikacji stron administracji publicznej? Czy w ogóle te analizy nie powinny być zintegrowane z unifikacją stron ministerstw?